python/arcticdb/version_store/_store.py

```diff
@@ -367,16 +367,14 @@ def _try_normalize(
             raise ArcticDbNotYetImplemented(
                 f"Not supported: normalizing\n"
                 f"symbol: {symbol}\n"
-                f"data:\n"
-                f"{dataframe}\n"
                 f"metadata:\n"
                 f"{metadata}\n"
```

Comment:
Drop the metadata too? Its string representation could also be large.
Drop the newline characters while we're here? Multi-line logging is a bad thing (hard to grep, interleaving problems, lines without timestamps)